docs/count-lint-violations.sh

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env bash
+
+# Outputs a sorted list of lint violations by rule.
+# Inspired by this: https://macr.ae/article/counting-eslint-disabled
+
+# Example output:
+#
+# DISABLED ESLINT RULES:
+#  40 ember/closure-actions
+#  35 ember/no-jquery
+#  33 ember/no-observers
+#   5 complexity
+#   2 no-console
+# DISABLED EMBER-TEMPLATE-LINT RULES:
+#  21 no-bare-strings
+#   4 require-valid-alt-text
+#   4 no-inline-styles
+#   3 table-groups
+# DISABLED STYLELINT RULES:
+#  14 declaration-no-important
+#   2 length-zero-no-unit
+
+if [ "$#" -eq 0 ]; then
+    echo "You must enter at least one command line argument with the path to search."
+    exit 1
+fi
+
+# These regexps are used to remove all characters and whitespace surrounding the rule names in the disable directive comment.
+
+# /* eslint-disable-line no-console */
+# /* eslint-disable-line no-console -- explanation of why it is disabled */
+# // eslint-disable-line no-console, no-empty
+REGEXP_C_STYLE_COMMENT_SHELL='^.*\/[*\/] +| +\*\/|, *| +--.+| +$'
+
+# {{! template-lint-disable no-bare-strings no-negated-condition }}
+# {{!-- template-lint-disable no-bare-strings --}}
+REGEXP_HBS_STYLE_COMMENT_SHELL='^.*{{!-{0,2} *| *-{0,2}}}.*$'
```

Comment:
This is only going to trigger if there's exactly one space. Is that a requirement for the comment to work?

Reply:
I'll tweak this to allow multiple spaces and add tests when I convert this to a real script.